src/environments/common.ts

```diff
@@ -0,0 +1,18 @@
+// Common values for all environments
+// You can override those in environment configuration files
+
+export const common = {
+  contactAddress: 'contact@zonemaster.net',
+  logoUrl: 'assets/images/zonemaster_logo_2021_color.png',
+  clientInfo: {version: '3.3.0', id: 'Zonemaster GUI'},
```

Comment:
The `clientInfo` should not be possible to override. That is the version and identity of the code. The rest is fine.

Reply:
done